packages/client/components/Avatar/Avatar.tsx

```diff
@@ -3,14 +3,18 @@ import React, {forwardRef} from 'react'
 import defaultUserAvatar from '../../styles/theme/images/avatar-user.svg'
 import AvatarBadge from '../AvatarBadge/AvatarBadge'
 
-type ImageBlockProps = Pick<Props, 'sansRadius' | 'sansShadow' | 'picture' | 'size' | 'onClick'>
+type ImageBlockProps = Pick<
+  Props,
+  'sansRadius' | 'sansShadow' | 'hasBorder' | 'picture' | 'size' | 'onClick'
+>
 
 const ImageBlock = styled('div')<ImageBlockProps>(
-  ({sansRadius, sansShadow, picture, size, onClick}) => ({
-    backgroundImage: `url(${picture}), url(${defaultUserAvatar})`,
+  ({sansRadius, sansShadow, hasBorder, picture, size, onClick}) => ({
```

Comment:
The `hasBorder` prop is used to indicate whether the profile image should retain its border.